drizzlepac/wfpc2Data.py

```diff
@@ -429,3 +430,129 @@ def wfpc2_to_flt(imgname):
         in_dq.close()
 
     return flt_filename
+
+
+# ------------------------------------------------------
+# Function for updating headers to latest
+# reference files from CRDS
+# ------------------------------------------------------
+def apply_bestrefs(raw_filename=None, dirname=None, uref_path=None, crds_path=None):
+    """Update WFPC2 data to use the latest reference files from CRDS
+
+    .. note::
+        See `https://hst-crds.stsci.edu/docs/cmdline_bestrefs/
+        <https://hst-crds.stsci.edu/docs/cmdline_bestrefs/>`_
+        for details on how to configure CRDS for your local system and for definitions
+        of all the environment variables used by CRDS.
+
+    Parameters
+    ----------
+    raw_filename : str, optional
+        Filename of RAW (*d0m.fits) input file to be updated
+        The corresponding calibrated (*c0m.fits) file also needs to be
+        present in the directory to be updated as well.
+        If not specified, **all RAW and calibrated files** from the
+        current directory, or ``dirname`` directory if given, will
+        be updated.
+
+    dirname : str, optional
+        Name of directory containing WFPC2 data to be updated.
+        If not specified, current directory will be checked.
+
+    uref_path : str
+        Path for ``uref`` directory on local system.
+
+    crds_path : str
+        Path for the ``CRDS_PATH`` directory on local system.
+        .. warning:: If no value is specified and ``CRDS_PATH`` is not
+        already defined locally, this function will create a temporary
+        ``CRDS_PATH`` directory tree under the directory with the files
+        to be processed, populate it with the latest reference files
+        and mappings needed by CRDS, then delete it when done.
+
+    """
+    starting_dir = os.getcwd()
+    import glob
```

Comment:
This import be at the top of the module.

Reply:
Thanks for catching that!  Removed in 74dd1d7.